src/components/wrappers/ProcessWrapper.tsx

```diff
@@ -148,9 +149,11 @@ export const ProcessWrapper = () => {
             path='*'
             element={
               <PDFWrapper>
-                <PresentationComponent type={ProcessTaskType.Data}>
-                  <Form />
-                </PresentationComponent>
+                <PaymentProvider>
```

Comment:
This could probably be placed late in `<FormProvider>` instead, otherwise the Payment component will now crash the app if placed in:
- Stateless forms
- Custom receipts (!)
- Sub forms

None of these are probably likely use-cases, but it's better to validate that in the component than having the whole app crash because of it. FormProvider is 'the place' for all providers needed by form components, at least.